src/ui/CategoryFacet/CategoryFacetHeader.ts

```diff
@@ -0,0 +1,73 @@
+import { CategoryFacet } from './CategoryFacet';
+import { $$ } from '../../utils/Dom';
+import { l } from '../../strings/Strings';
+import { SVGIcons } from '../../utils/SVGIcons';
+import { SVGDom } from '../../utils/SVGDom';
+import { AccessibleButton } from '../../utils/AccessibleButton';
+
+export interface ICategoryFacetHeaderOptions {
+  categoryFacet: CategoryFacet;
+  title: string;
+}
+
+export class CategoryFacetHeader {
+  public element: HTMLElement;
+  public eraserElement: HTMLElement;
```

Comment:
Would it be possible to encapsulate these variables such as `eraserElement` and `waitElement`? Also on line 18, could the `options` variable be private?